src/map/packets/basic.h

```diff
@@ -180,6 +181,16 @@ class CBasicPacket
     {
         return reinterpret_cast<int8*>(data) + index;
     }
+
+    // used for setting "proper" packet sizes rounded to the nearest four away from zero
+    uint32 roundUpToNearestFour(uint32 input)
+    {
+        int remainder = input % 4;
+        if (remainder == 0)
```

Comment:
Needs some {}'s, but good helper.
Can we get the 2, 4, and 8 siblings of this please?

Reply:
Maybe move to utils

Reply:
Rounding up to the nearest 4 is a feature of how SE aligns packets, so I thought it made sense to put it in the packet class

Reply:
Added the brackets

Reply:
Sure, if this was a member method. But since this takes in and outputs a uint32, it isn't specific to packet logic, it's just a number thing

Reply:
It already is a member method of CBasicPacket and won't be used anywhere else but in packets? I'm not really following